06.bowling_object/game.rb

```diff
@@ -0,0 +1,38 @@
+# frozen_string_literal: true
+
+require './frame'
+require './shot'
+
+class Game
+  def initialize(pinfall_text)
+    @frames = parse_pinfall_text(pinfall_text).each_with_index.map { |shots, index| Frame.new(index, shots) }
+  end
+
+  def main
+    total_score = @frames.sum { |frame| frame.calculate_score(@frames) }
+    puts total_score
+  end
+
+  private
+
+  def parse_pinfall_text(pinfall_text)
+    all_shots = pinfall_text.split(',').map { |shot| Shot.new(shot) }
+    shots = []
+    shots_each_frame = []
+    all_shots.each do |shot|
```

Comment:
```suggestion
    all_shots.each_with_object([]) do |shot, shots_each_frame|
```

each_with_objectが使えるかも